.github/workflows/perform-release.yml

```diff
@@ -137,7 +137,7 @@ jobs:
           # Wait for the deploy-artifacts.yml workflow to start
           sleep 3
```

Comment:
Is it really safe?

Reply:
Not ideal, but it does the job, at least until https://github.com/cli/cli/issues/4001 is implemented